nemo_text_processing/inverse_text_normalization/en/taggers/date.py

```diff
@@ -93,11 +102,31 @@ def _get_thousands_graph():
     graph_ties = _get_ties_graph()
     graph_digits = _get_digits_graph()
     graph_thousands = _get_thousands_graph()
+    graph_ad_bc = (
+        (pynini.cross(" a d", ' AD') | pynini.cross(" b c", ' BC'))
+        | (pynini.cross(" anno domini", ' AD') | pynini.cross(" before christ", ' BC'))
+        | (pynini.cross(" c e", ' CE') | pynini.cross(" b c e", ' BCE'))
+        | (pynini.cross(" common era", ' CE') | pynini.cross(" before common era", ' BCE'))
+    )
```

Comment:
could you move these words to files, please? when adding ITN for ASR+PC model it'll make integration easier

Reply:
Moved to `year_suffix.tsv`.=